test/test_optim.py

```diff
@@ -596,7 +596,7 @@ def test_complex_2d(self, device, dtype, optim_info):
                 # For other pointwise optimizers, this distinction is trivial, but for LBFGS where
                 # there are reductions across all parameters (and all the grads get flattened into
                 # one long Tensor), this ordering matters. Why? Reductions (like sum) are NOT
-                # commutative, i.e., a + b + c != a + c + b in computers. Thus, we add a seed here
+                # associative, i.e., a + b + c != a + c + b in computers. Thus, we add a seed here
```

Comment:
"Reductions are not deterministic because addition between floating point numbers is not associative"